csit-sut/Dockerfile

```diff
@@ -1,4 +1,5 @@
 # Copyright (c) 2019 Cisco and/or its affiliates.
+# Copyright (c) 2019 PANTHEON.tech s.r.o. and/or its affiliates.
```

Comment:
can't eat it with this....flip it back to cisco and i'll merge it straight away

Reply:
Removed.

After you merge it, could you build and push the images? And also setup csit-shim on the vpp device server?